scripts/modules.sh

```diff
@@ -1,7 +1,7 @@
 . ./modules
 [ ! -f "$GLUON_SITEDIR"/modules ] || . "$GLUON_SITEDIR"/modules
 
-FEEDS="$(echo $GLUON_FEEDS $GLUON_SITE_FEEDS | tr ' ' '\n')"
+FEEDS="$(echo "$GLUON_FEEDS" "$GLUON_SITE_FEEDS" | tr ' ' '\n')"
```

Comment:
Not using quotes here is preferable, as this will also drop extraneous whitespace.